storage/Utils/AppUtil.cc

```diff
@@ -368,13 +374,33 @@ bool isNfsDev( const string& dev )
 	    {
 		string::size_type pos2 = content.find('\n', pos1);;
 		if (pos2 != string::npos || pos1 != content.length())
-		    pfc(level, component, file, line, func,
-			content.substr(pos1, pos2 - pos1));
+		    logger->write(level, component, file, line, func,
+				  content.substr(pos1, pos2 - pos1));
 		if (pos2 == string::npos)
 		    break;
 		pos1 = pos2 + 1;
 	    }
 	}
+	else
+	{
+	    // legacy
+	    CallbackLogDo pfc = getLogDoCallback();
```

Comment:
pfc? Private First Class? ;-)

Maybe use something more speaking like, say, app_logger_callback() ? it's only used twice in the function, and that name change would make quite some difference when trying to understand the code.